app/policies/work_policy.rb

```diff
@@ -22,10 +22,8 @@ def create_version?
     editable? && record.draft_version.blank?
   end
 
-  # @note we are temporarily disabling DOIs on draft works for launch. This
-  # method should probably go away when we re-enable them
   def mint_doi?
```

Comment:
This authorization logic applies [here](https://github.com/psu-libraries/scholarsphere/blob/fc8f4848bd29e7eae6cb1ec9f352cfeae83ec29c/app/views/dashboard/works/edit.html.erb#L124) and [here](https://github.com/psu-libraries/scholarsphere/blob/fc8f4848bd29e7eae6cb1ec9f352cfeae83ec29c/app/views/dashboard/collections/edit.html.erb#L49) to determine whether or not to show the UI for minting a DOI, but it looks like different authorization logic is being applied [here](https://github.com/psu-libraries/scholarsphere/blob/fc8f4848bd29e7eae6cb1ec9f352cfeae83ec29c/app/controllers/dois_controller.rb#L9) to determine whether or not a DOI is actually allowed to be minted, and I'm not sure why that is. It seems to me that the controller should
```ruby
authorize(resource, :mint_doi?)
```
also, but it's more lax.

Does anybody know why that is? @rschenk ?

Reply:
I assume that it has something to do with the stale comment that I deleted above...

Reply:
I don't remember why we made this change for only published works to get DOIs, but I do seem to remember it was an afterthought or workaround for something. 

My guess is the controller is more lax because either we forgot, or  because things other than Works can get DOIs and for some insane reason we didn't want to go back and update them all? Frankly I have no idea.

Reply:
Just adding a comment to Ryan's note. Both works and collections can get DOIs

Reply:
Alright - I'm just going to leave well enough alone here. Since there doesn't seem to be any problem with how the controller is working, I'll keep it the way that it is.